media/website/content/migration.md

```diff
@@ -30,6 +30,21 @@ Auditor REST APIs are changed as shown in the table below.
 
 `/records` endpoint handles multiple and bulk record operations such as inserting bulk records and querying multiple records.
 
+## Apel plugin
+
+The config parameter `site_name_mapping` is removed and the structure of the config parameter `sites_to_report` is changed. `sites_to_report` is now a dictionary, where the keys are the site names as configured in the GOCDB, and the values are lists of the corresponding site names in the AUDITOR records.
+
+Before:
+```
```

Comment:
Maybe indicate that this is python so that we get proper syntax highlighting:

```
```python
```

Reply:
ok